cmd/fyne/internal/commands/build.go

```diff
@@ -67,6 +69,17 @@ func Build() *cli.Command {
 				Usage:       "Specify a name for the output file, default is based on the current directory.",
 				Destination: &b.target,
 			},
+			&cli.BoolFlag{
+				Name:        "pprof",
+				Usage:       "Enable pprof profiling.",
+				Destination: &b.pprof,
+			},
+			&cli.IntFlag{
+				Name:        "pprof-port",
+				Usage:       "Specify the port to use for pprof profiling.",
+				Value:       6060,
```

Comment:
Is 6060 a normal default?
We use 8080 in "fyne serve", shouldn't we be in the same range for all our port defaults?

Reply:
Yes, it is. That's the port that all net pprof example use: https://pkg.go.dev/net/http/pprof